about.html

```diff
@@ -0,0 +1,165 @@
+            <a href="./about.html">
+              <li>About</li>
+            </a>
+            <a href="">
+              <li>Sponsor</li>
+            </a>
+            <a href="">
+              <li>News</li>
+            </a>
+            <a href=""><button type="button">Join Now</button></a>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <div id="mobile-menu">
+      <div id="close">
+        <i class="fa-solid fa-xmark"></i>
+      </div>
+      <ul>
+        <li><a id="homelink" href="index.html"> Home</a></li>
+        <li><a id="aboutlink" href="about.html"> About</a></li>
+        <li><a id="sponsorlink" href="#"> Sponsor</a></li>
+        <li><a id="newslink" href="#"> News</a></li>
+        <li><a id="join-us" href="#"> Join Us</a></li>
+      </ul>
+    </div>
+    <!-- intro -->
+    <section class="intro">
+      <button id="hamburgerBtn">
+        <i class="fa-solid fa-bars"></i>
+      </button>
+      <div class="intro-top">
+        <p>"Hello! Sharing World"</p>
+        <h1>
+          SINOLOGY COMMONS<br />
+          GLOBAL SUMMIT 2015
+        </h1>
+      </div>
+      <p class="intro-mid">
+        A joyful Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla
+        architecto omnis vitae deleniti nostrum necessitatibus? Hic sunt minus
+        quos impedit modi nam nemo?
+      </p>
+
+      <div class="intro-bottom">
+        <p>
+          Please contact us per Email for any further questions about CC Global
+          Summit 2015!
+        </p>
+        <aside><strong>scsummit2015@scchina.org</strong></aside>
+      </div>
+    </section>
+
+    <!--current summits-->
+    <section class="current">
+      <h3>Current Summits<br />Global Summit 2015 Logo<br /></h3>
+      <hr />
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci quo
+        rem ipsum maxime! Omnis, fugit. Lorem ipsum dolor sit, amet consectetur
+        adipisicing elit. Quas recusandae illo laborum eaque? Voluptatem, saepe?
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+      </p>
+      <div>
+        <img src="images/current/door.jpg" />
+        <h5>Enter the <span>2023</span> Field S<span>umm</span>it</h5>
+      </div>
+
+      <div><hr /></div>
+    </section>
+    <!--past summits-->
+    <section class="past">
+      <!-- <div class="past-container"> -->
+      <h3>See the past SC Summits<br /></h3>
+      <hr />
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur.
+      </p>
+      <div class="past-container">
+        <div class="past-item-1">
+          <img src="images/past-events/shanghai.jpg" />
+          <h2>2013</h2>
+          <p>SC Global Summit 2013 in Shanghai</p>
+        </div>
+
+        <div class="past-item-2">
+          <img src="images/past-events/beijing.jpg" />
+          <h2>2011</h2>
+          <p>SC Global Summit 2013 in Beijing</p>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly position the `summit cards` correctly on your `about` page as they appear to overflow towards the footer section blocking its content.

![image](https://user-images.githubusercontent.com/98453979/218112118-b4e345f0-2ac0-4d52-aca6-bc6ae6923385.png)

- [ ] This also affected the mobile version of your page

![image](https://user-images.githubusercontent.com/98453979/218112636-26595983-1285-421d-b9b7-1ad649c858e8.png)